corehq/apps/app_manager/suite_xml/sections/entries.py

```diff
@@ -316,7 +316,8 @@ def entry_for_module(self, module):
                     detail_confirm=detail_confirm,
                     detail_persistent=self.get_detail_persistent_attr(module, module, "case_short"),
                     detail_inline=detail_inline,
-                    autoselect=module.auto_select_case,
+                    autoselect=module.is_auto_select(),
+                    max_select_value=module.max_select_value,
```

Comment:
Nitpick: it seems a little distracting to have this attribute in the suite for datums that don't need it. What would you think of only adding it if autoselect is true?

Reply:
Agreed, but  believe it already filters out somewhere else in the code if false. This is a test result example with autoselect = false since the default is false, but autoselect is not in the suite: [here](https://github.com/dimagi/commcare-hq/pull/32304/files#diff-dda090072f034f152b598cba7e7e48ec2b55101b35ef06b37bf9fbacf34c0ea3L75 )

Reply:
Are we talking about the same thing? Looking at that link, what I mean to say is that I don't think `basic_remote_request.xml` should have changed, I think it's confusing that its datum doesn't use autoselect but does include `max-select-value`.

Reply:
Ah I see we were not talking about the same thing! So you're referring to `max-select-value` yes that is included each time (I believe correct me if thats still not right). I have that included since that would apply to all multi-select case lists regardless of auto-select to also set the max-select-value for manual selections (which is confusing given the purpose of this PR).

Reply:
aha, I had indeed forgotten that the max value also applies to non-auto-select, thank you for explaining